### ISSUE TITLE
ExpandedEmergency data is not available in generated proto files

### ISSUE THREAD

**Issue by sdupont-jive** [googlei18n/libphonenumber#1994]
Hi,

From [ShortNumberInfo.java](https://github.com/googlei18n/libphonenumber/blob/master/java/libphonenumber/src/com/google/i18n/phonenumbers/ShortNumberInfo.java) I want to test for expandedEmergency numbers defined in [ShortNumberMetadata.xml](https://github.com/googlei18n/libphonenumber/blob/master/resources/ShortNumberMetadata.xml). It does not seem possible.

After digging, it seems the expandedEmergency data does not make it in the [proto files](https://github.com/googlei18n/libphonenumber/tree/master/java/libphonenumber/src/com/google/i18n/phonenumbers/data) generated by the [build short metadata tool](https://github.com/googlei18n/libphonenumber/blob/master/java/build.xml#L48-L60).
Also it does not appear in [phonemetadata.proto](https://github.com/googlei18n/libphonenumber/blob/master/resources/phonemetadata.proto#L154)
Is there a particular reason for this omission? I searched in the list of issues, looked at the release notes, etc and did not find anything.
From what I see, adding expandedEmergency data to the proto files should be relatively straight forward but I have the feeling I am missing something.

**Comment by andyst** [comment]
Indeed, the <expandedEmergency> metadata is not exposed through APIs and thus not serialized / embedded in the library's runtime metadata.

At some point we started collecting <expandedEmergency> metadata, but inconsistently across countries. There was no strong interest / use-case, so we stopped adding more data.

Note: Not to be confused  with <emergency> which is maintained very well and a critical use-case of the library.

At this point we don't feel comfortable exposing the <expandedEmergency>. Fleshing it out (clear specs for the metadata and APIs) would be needed first.

Does this answer your question?

**Comment by sdupont-jive** [comment]
Yes you did answer my question. 
I was under the impression the `<expandedEmergency>` had the same level of attention as `<emergency>`.

Thank you.
